<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavingNotificationDialog</class>
 <widget class="QDialog" name="SavingNotificationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">SaveFile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/appicon/48.png</normaloff>:/appicon/48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="100,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,100" columnstretch="0,80">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_Title">
        <property name="text">
         <string notr="true">Template: SETTINGS</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="userItems"/>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="icon">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../_resources/editor.qrc">:/dialogues/warn.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Discard">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Cancle">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
